tests/localosmosis/scripts/submit_upgrade_proposal.sh

```diff
@@ -58,6 +58,22 @@ make_proposal() {
         --yes \
         -o json"
 
+    OSMOSIS_CMD="osmosisd tx gov submit-proposal software-upgrade \
```

Comment:
Why is this copied? Same thing is right above

Reply:
Ah this was from fixing merge conflict, solved